CHANGELOG

```diff
@@ -1,3 +1,6 @@
+To be released
+    - Add background-color mixin
+    - Removed `base-bg-color`, `bg-color-neutral-30`, `bg-color-error`, and `bg-color-light-error` mixins
```

Comment:
Sorry for running you in circles on this one, but with your most recent commit, this point should now read `deprecated` instead of `removed`